rastervision/runner/aws_batch_experiment_runner.py

```diff
@@ -75,13 +86,40 @@ def batch_submit(self,
         if parent_job_ids is None:
             parent_job_ids = []
 
-        full_command = command.split()
+        if command_type == DEPENDENCY_GROUP_JOB:
+            full_command = NOOP_COMMAND.split()
+        else:
+            full_command = command.split()
 
         client = boto3.client('batch')
 
         uuid_part = str(uuid.uuid4()).split('-')[0]
         exp = ''.join(e for e in experiment_id if e.isalnum())
-        job_name = '{}_{}_{}'.format(command_type, exp, uuid_part)
+        job_name = '{}-{}_{}_{}'.format(command_type, command_split_id, exp,
+                                        uuid_part)
+
+        group_level = 1
```

Comment:
@lewfish commented in slack:

You can use Batch "array jobs" for this. https://docs.aws.amazon.com/batch/latest/userguide/array_jobs.html There's some code for this here https://github.com/azavea/raster-vision/blob/5ce55eef38abfc774779a9c93ccadca7e905d73a/rastervision/runner/aws_batch_experiment_runner.py#L107

Reply:
We'd have to figure out how to utilize AWS_BATCH_JOB_ARRAY_INDEX inside `run_command`, as this is the only mechanism for splitting up the command parameters AFAIK. https://github.com/azavea/raster-vision/blob/f62810a99f0c5f00e1467163130e01ea05c757ea/rastervision/cli/main.py#L247

We could feed an option into run_command that specifies the environment variable that defines the split ID, and then modify the command config appropriately. Thoughts?

Reply:
This will also affect the mechanism for submission - the submission happens a command at a time, and does not have the visibility of e.g. all CHIP commands at once. This may take a substantial refactor to work out - and may take some reconsideration of the OutOfProcessRunner heirarchy.

Perhaps we should do this in a separate PR?

Reply:
Yeah, that would be a big change. I think the way you have it is ok.